spec/lib/zendesk_apps_tools/cache_spec.rb

```diff
@@ -11,8 +11,8 @@
     let(:zat_file) { File.join(tmpdir, '.zat') }
 
     before do
-      content = JSON.dump(subdomain: 'under-the-domain', username: 'Roger', app_id: 12)
-      File.write(zat_file, content, mode: 'w')
+      content = JSON.dump(subdomain: 'under-the-domain', username: 'Roger@something.com', app_id: 12)
+      File.write(File.join(tmpdir, '.zat'), content, mode: 'w')
```

Comment:
This doesn't need to be changed I don't think?

Reply:
Yeap. Glad we caught this.